lib/index.js

```diff
@@ -79,9 +68,14 @@ const init = async function (options = {}) {
   }
 
   const server = await Steerage.init({
-    config: Path.join(__dirname, 'config.json'),
+    basedir: baseDir,
+    config: [
+      Path.join(__dirname, 'config.json'),
+      ...(userConfigPath || [])
```

Comment:
can a default of `[]` be provided in the joi schema for `userConfigPath`?

Reply:
ah, yes 👍